code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -123,6 +125,12 @@
 	burnt = 1
 	update_icon()
 
+/turf/open/floor/fakepit	
+	name = "chasm"
```

Comment:
there is already a tell. plus you can just toss a pen or something on it to check.